### ISSUE TITLE
[bug] when devstats score is equal for two github ids, the ranks are sorted in reverse order

### ISSUE THREAD

**Issue by kfaseela** [cncf/devstats#40]
Please see the attached screenshot for the problem
![devstats-bug](https://github.com/cncf/devstats/assets/7901446/6e1fa40e-4737-41ff-a769-3ad4fe71ac02)

**Comment by lukaszgryglicki** [comment]
I will TAL tomorrow, but this is rather about sorting by more than 1 column, from what I know you can sort by a given column in Grafana, and then other columns have random order for rows with the same value for "sort order" column - but, anyway, will inmvestigate what I can do probably tomorrow or on Monday.

**Comment by kfaseela** [comment]
would be nice if atleast the rank can be shown in ascending order. showing 15 first and 14 next does not look great

**Comment by lukaszgryglicki** [comment]
Sure, rank should actually be the same then...

**Comment by lukaszgryglicki** [comment]
This is `WIP` atm.

**Comment by lukaszgryglicki** [comment]
Fixed, I've updated the condition for calculating Rank value - this counts by `value desc` and then `username/login asc` (for the same value in 1st condition) - finally dashboards will be sorted by `Rank asc` instead of `value desc`.

This changes 9 dashboards on 170+ projects, so it will take some time to repopulate.
Coding is done and all dashboards are being reloaded now (just thir grafana instances) - will update once more when this is done.
cc @kfaseela
